pkg/test/dualstack/cloud_providers_dualstack_test.go

```diff
@@ -131,11 +131,13 @@ func TestCloudClusterIPFamily(t *testing.T) {
 			cloudName: "aws",
 			osNames: []string{
 				"ubuntu",
+				"centos",
 			},
-			cni:                 "canal",
-			ipFamily:            util.DualStack,
-			skipNodes:           true,
-			skipHostNetworkPods: true,
+			cni:                    "canal",
+			ipFamily:               util.DualStack,
+			skipNodes:              true,
+			skipHostNetworkPods:    true,
+			skipEgressConnectivity: true,
```

Comment:
Why egress connectivity is not working on AWS, if it was before?

Reply:
Egress doesn't work for CentOS on AWS. 

But this change disables them for both. That is not good! :grimacing: 

Lemme check whats up there.